src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -6514,6 +6523,15 @@ public IContentWriter GetContentWriter(string path)
 
             try
             {
+                // Throw error if path is a directory.
+                if (Directory.Exists(path))
+                {
+                    string errMsg = StringUtil.Format(SessionStateStrings.GetContainerContentException, path);
```

Comment:
We need another message for "write" cmdlets.

Reply:
Added different error message for Write to container exception.